packages/docs/docs/getting-started/layouts.mdx

```diff
@@ -70,6 +70,33 @@ class has additional `size` and `offset` properties:
 <ApiSnippet url={'/api/2d/components/Layout#offset'} />
 <hr />
 
+## Opting Out of Layout 
+
+You can disable the layout behaviour by passing `layout={false}` to your component:
+
+```tsx
+<Layout layout direction="row">
+  <Rect width={100} height={100} fill="green"/>
+  <Rect width={100} height={100} fill="red" layout={false} x={200} y={50}/>
+</Layout>
+```
+
+This behaves almost similar to **absolute postioning** in CSS Flexbox: 
+- The node doesn't affect or get affected by layout flow
+- You're responsible for positioning it manually
+- It can still be visually inside a layout container
+
+```css
+.parent{
+  display:flex;
+}
+
+.child{
+  position: absolute;
+}
+```
+
+You can learn more about such layout exceptions in the [Groups](#groups) section below.
```

Comment:
Space between sections. Plus a small tweak for concision.
```suggestion
You can learn more about layout exceptions in the [Groups](#groups) section below.

```

As an aside, it might make sense to group the Group section and this one together so that one looking for the exceptions can find both at once.